docs/api.md

```diff
@@ -0,0 +1,368 @@
+# Overview
+dYdX offers a few HTTP APIs for retrieving information about the protocol, and submitting orders to
+our DEX. Feel free to use these APIs to build your own applications on top of dYdX. Please feel
+free to let us know via Intercom or Telegram if you have any questions or experience any issues.
+
+All of these endpoints live at `https://api.dydx.exchange/`
+
+e.g. `https://api.dyd.exchange/v1/orders`
+
+## Orderbook
+
+### Introduction
+
+The following API endpoints allow for submitting and retrieving orders from the dYdX orderbook.
+This orderbook is what's frequently referred to as a "Matching Model" orderbook. This means that
+all orders are submitted to the blockchain by dYdX itself You do not need to provide gas fees
+or send on-chain transactions yourself. This is especially useful for traders and market makers who
+wish to be able to quickly cancel their orders without waiting for a transaction to be mined.
+
+In order to submit an order, you (the maker) must first create a JSON object that specifies the
+details of your order. Once you create this object you must sign it with your Ethereum private key,
+and put the result in the `typedSignature` field. Note: The `typedSignature` is omitted before
+signing, and added only after signing the message.
+
+After this is done, the order is ready to be submitted to the API.
+
+__Order fields__
+
+|Field Name|JSON type|Description|
+|----------|---------|-----------|
+|makerMarket|string|The Solo [market](https://docs.dydx.exchange/#/overview?id=markets) of the Maker amount|
+|takerMarket|string|The Solo [market](https://docs.dydx.exchange/#/overview?id=markets) of the Taker amount|
+|makerAmount|string|The amount of token the Maker is offering in base units|
+|takerAmount|string|The amount of token the Maker is requesting from the taker base units|
+|makerAccountOwner|string|The Ethereum address of the Maker.|
+|takerAccountOwner|string|The Ethereum address of the Taker.|
```

Comment:
add a note that for our API this must be set to our account (same for taker account number)

Reply:
Do we know what this owner/number is yet?